src/App.js

```diff
@@ -1,85 +1,129 @@
 import React from 'react';
+import TodoApp from './components/TodoApp/TodoApp';
+import TodoList from './components/TodoList/TodoList';
+import TodosFilter from './components/TodosFilter/TodosFilter';
 
-function App() {
-  return (
-    <section className="todoapp">
-      <header className="header">
-        <h1>todos</h1>
+class App extends React.Component {
+  state = {
+    todos: [],
+    toggleAll: false,
```

Comment:
`toggleAll` is a name for a function, use `isToggledAll` (as if you wanna answer yes-no)